Userland/Libraries/LibWeb/CSS/StyleValue.cpp

```diff
@@ -1955,6 +1952,175 @@ bool PositionValue::operator==(PositionValue const& other) const
         && variant_equals(vertical_position, other.vertical_position));
 }
 
+String RadialGradientStyleValue::to_string() const
+{
+    StringBuilder builder;
+    builder.appendff("radial-gradient({} "sv,
+        m_ending_shape == EndingShape::Circle ? "circle"sv : "ellipse"sv);
+
+    m_size.visit(
+        [&](Extent extent) {
+            builder.append([&] {
+                switch (extent) {
+                case Extent::ClosestCorner:
+                    return "closest-corner"sv;
+                case Extent::ClosestSide:
+                    return "closest-side"sv;
+                case Extent::FarthestCorner:
+                    return "farthest-corner"sv;
+                case Extent::FarthestSide:
+                    return "farthest-side"sv;
+                default:
+                    VERIFY_NOT_REACHED();
+                }
+            }());
+        },
+        [&](CircleSize const& circle_size) { builder.append(circle_size.radius.to_string()); },
+        [&](EllipseSize const& ellipse_size) { builder.appendff("{} {}", ellipse_size.radius_a.to_string(), ellipse_size.radius_b.to_string()); });
+
+    if (m_position != PositionValue::center()) {
+        builder.appendff(" at "sv);
+        m_position.serialize(builder);
+    }
+
+    builder.append(", "sv);
+    serialize_color_stop_list(builder, m_color_stop_list);
+    builder.append(')');
+    return builder.to_string();
+}
+
+Gfx::FloatSize RadialGradientStyleValue::resolve_size(Layout::Node const& node, Gfx::FloatPoint center, Gfx::FloatRect const& size) const
+{
+    auto side_shape = [&](auto distance_function) {
```

Comment:
You can `const` all lambdas here

Reply:
Do you mean making the declarations const? What's the benefit there, seems a little more verbose (it's not like I have anything I could assign to the lambda, and the parameters are passed by value).

Reply:
At least to follow the [Core guideline](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rconst-immutable).
We can also argue that it would help readability, as we can consider that it won't be moved, so used as a callback.
It might also have a performance impact.

Feel free to close the comment without adding them.

Reply:
I've done it anyway, but it won't have any performance impact. The compiler does not care about `const` declarations (since you can always cast the const away anyway).

Reply:
> The compiler does not care about const declarations

I don't think this is true.
Here is a notorious example: [CppCon video](https://youtu.be/zBkNBP00wJE?t=1614) (look at the two next minutes)

Moreover, `const_cast` are undefined behavior and definitely impacts generated code.